README.adoc

```diff
@@ -244,6 +252,14 @@ server:
   static_content_root_directory: VALUE
 ----
 
+|`PROMETHEUS_SERVICE_URL`
```

Comment:
PROMETHEUS_SERVICE_URL is not under `server`, it's at config root

Reply:
good catch...fixed.